### DISCUSSION TITLE
反応しなくなりました

### DISCUSSION THREAD

**Discussion by YOUYOU41** [Yowkees/keyball#363] [Q&A] [answered]
初めまして

一度完成して起動まで確認したのですがPCをつけるたびにリセットを押すかTRSケーブルを抜き差ししないと反応しなかったので
Pro Microの半田付けがいびつだからと思い修正しようとしたところGNDとその隣がくっついたので半田吸い取りで何度かやったら
写真のように焼けてしまいUSBを指してもリセットを押しても反応しなくなってしまいました。

１，つかなくなる前にやったことは
２，keyball61_via_v1.2.0を入れる
３，起動確認、入力も問題なし
４，はんだを修正
５，フリーズするようになる
６，TRSケーブルを抜き差しする
７，OLEDに何も表示されなくなる
８，半田を修正する
９，USBを刺しても何も表示されなくなる
こんな感じです

![IMG_4960](https://github.com/Yowkees/keyball/assets/146203069/93da44a8-b71f-4201-b5e3-082cfa67238c)
![IMG_4961](https://github.com/Yowkees/keyball/assets/146203069/c3ea4fb9-83a0-403a-8183-3b31247a7e8d)

基盤自体が壊れたのでしょうか？
キーボードを作るのは今回が初めてなもので修復は可能でしょうか？

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

ProMicro裏にレジストを剥がれた部分がありますが、この辺りには配線はなかったので大丈夫だと思います。
もしかすると、再半田の際に12Pピンソケット内に半田が流れ込みショートしているのかも知れません。

ProMicroを取り外し、ProMicro単体では起動するでしょうか？
また、ProMicroを取り外した状態で、ミドル基板のProMicro裏部分で隣とショートしている端子はあるでしょうか？（GNDどうしはショートしていて大丈夫です。）

以上、お手数ですがご確認お願いいたします。

**Reply by YOUYOU41** [reply]
教えていただいた通り
12ピンソケットを一度取り外したところショートしているような箇所があったため
再度付け直しをしたら無事起動することができました。
本当にありがとうございました。

**Comment by Yowkees** [comment]
無事に起動するようになり安心しました。
今後ともどうぞよろしくお願いいたします。
